.github/workflows/test-evm.yaml

```
@@ -119,10 +119,10 @@ jobs:
         run: node evm/deploy-its.js -m create2 -u -y
 
       - name: InterchainTokenService set hub
-        run: node evm/its.js --action setTrustedAddress --trustedChain axelar --trustedAddress axelar1xyz -y
+        run: node evm/its.js set-trusted-address axelar axelar1xyz -y
 
       - name: InterchainTokenService set trusted chain
-        run: node evm/its.js --action setTrustedAddress --trustedChain remote --trustedAddress hub -y
+        run: node evm/its.js set-trusted-address remote hub -y
```

```suggestion
        run: node evm/its.js add-trusted-chain remote -y
```
Use `add-trusted-chain` to set `hub` automatically if it's an amplifier chain (and ITS address set in the config if consensus chain, or if --trustedAddress override is provided) for consistent usage. Once we migrate EVM ITS from pairwise to hub for legacy consensus chains, then trusted address value can be removed entirely